.gitlab-ci.yml

```diff
@@ -44,27 +44,41 @@ test:
   stage: test
   tags:
     - kube
-  script:
+  allow_failure: false
```

Comment:
allow_failure "false" is the default for gitlab ci, so you can omit this statement